experiments/multichain/processing/DatabaseReader.py

```diff
@@ -0,0 +1,260 @@
+import os
+from os import path
+import base64
+
+import networkx as nx
+
+from Tribler.community.multichain.database import MultiChainDB
+from Tribler.community.multichain.payload import EMPTY_HASH
+from Tribler.community.multichain.community import GENESIS_ID
+from Tribler.community.multichain.conversion import PK_LENGTH
+
+EMPTY_HASH_ENCODED = base64.encodestring(EMPTY_HASH)
+GENESIS_ID_ENCODED = base64.encodestring(GENESIS_ID)
+
+
+class MultiChainExperimentAnalysisDatabase(MultiChainDB):
+    """
+    Extended MultiChainDB that provides additional functionality to analyze an experiment.
+    """
+
+    def __init__(self, dispersy, working_directory):
+        super(MultiChainExperimentAnalysisDatabase, self).__init__(dispersy, working_directory)
+
+    def get_totals(self, mid):
+        """
+        Return the totals in order by sequence number for mid.
+        """
+        mid = buffer(mid)
+        db_query = u"SELECT total_up, total_down FROM(" \
+                   u"SELECT total_up_responder as total_up, total_down_responder as total_down, sequence_number_responder " \
+                   u"as sequence_number FROM multi_chain WHERE mid_responder == ? AND sequence_number_responder != -1 "\
+                   u"UNION " \
+                   u"SELECT total_up_requester as total_up, total_down_requester " \
+                   u"as total_down, sequence_number_requester as sequence_number FROM multi_chain WHERE mid_requester =?) " \
+                   u"ORDER BY sequence_number"
+        db_result = self.execute(db_query, (mid, mid)).fetchall()
+        return db_result
+
+
+class DatabaseReader(object):
+
+    def __init__(self, working_directory):
+        """
+        Parent class that contains functionality for both gumby and single database readers.
+        DatabaseReaders process experimentation information.
+        """
+        """ Either contains all information all ready or all information will be aggregated here."""
```

Comment:
I noticed this also in other commits. In general, we are using comments starting with # (hashtag) if it's only one line. Please change this so it follows our style.